rpp30-boc-meltdown/biteshare
# Split bill options

## Description

Added the `split bill methods`  section for host view on the summary page.
Also added a placeholder guest list component which will be implemented in another PR.

## How to test

Pull this branch, run the bundle, and verify that the screen looks like this:
![Screen Shot 2021-12-21 at 6 31 38 PM](https://user-images.githubusercontent.com/43324065/147154083-56af0a2d-2c7d-4fa6-88ab-1d829ab5ee64.png)

Click on `Button`, the style for this section would change to inform the host to choose between a method:
![Screen Shot 2021-12-21 at 6 31 47 PM](https://user-images.githubusercontent.com/43324065/147154297-de76939a-e6ea-4f95-8061-737ad7cebe07.png)

Comment out `<DummyComponent />` and it will look like this:
![Screen Shot 2021-12-21 at 6 18 36 PM](https://user-images.githubusercontent.com/43324065/147154158-8ed4b846-a59b-4a24-8941-2cb5128a3622.png)


## Notes



## Issue tracking

https://trello.com/c/KyJX2aai
